### PR TITLE
(BREAKING) Update i18next interpolation syntax

### PR DESCRIPTION
**Pull request by vasharma05**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [x] My work includes tests or is validated by existing tests.

## Summary
This PR is a child PR of https://github.com/openmrs/openmrs-esm-core/pull/799.

Notable changes in translations:
1. All the variables should be wrapped in `{{` and `}}` instead of `{` and `}`.
2. Use `count` variable to leverage the pluralization support.

### Dev3 after merging https://github.com/openmrs/openmrs-esm-core/pull/799 but without this PR
image

### These changes along with changes in Dev3
image

## Related Issue
None

## Other
This PR also includes update to the translation for the pagination component in the common-lib.

### CODE REVIEW

File: packages/esm-patient-banner-app/translations/en.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
   "patientLists": "Patient Lists",
   "postalCode": "Postal code",
   "relationships": "Relationships",
-  "seeMoreLists": "See {listCount} more lists",
+  "seeMoreLists_one": "See {{count}} more lists",
```
Comment (denniskigen) [inline packages/esm-patient-banner-app/translations/en.json:22]:
```suggestion
  "seeMoreLists_one": "See {{count}} more list",
```

File: packages/esm-patient-banner-app/translations/fr.json [hunk 2]
```diff
@@ -19,7 +19,9 @@
   "patientLists": "Patient Lists",
   "postalCode": "Postal code",
   "relationships": "Relations",
-  "seeMoreLists": "See {listCount} more lists",
+  "seeMoreLists_one": "See {{count}} more lists",
```
Comment (denniskigen) [inline packages/esm-patient-banner-app/translations/fr.json:22]:
```suggestion
  "seeMoreLists_one": "See {{count}} more list",
```

File: packages/esm-patient-banner-app/translations/he.json [hunk 3]
```diff
@@ -19,7 +19,10 @@
   "patientLists": "Patient Lists",
   "postalCode": "מיקוד",
   "relationships": "יחסים",
-  "seeMoreLists": "See {listCount} more lists",
+  "seeMoreLists_one": "See {{count}} more lists",
```
Comment (denniskigen) [inline packages/esm-patient-banner-app/translations/he.json:22]:
```suggestion
  "seeMoreLists_one": "See {{count}} more list",
```

File: packages/esm-patient-chart-app/src/index.ts [hunk 4]
```diff
@@ -17,7 +17,7 @@ import { genericNavGroupConfigSchema } from './side-nav/generic-nav-group.compon
 
 // This allows @openmrs/esm-patient-common-lib to be accessed by modules that are not
 // using webpack. This is used for ngx-formentry.
-window["_openmrs_esm_patient_common_lib"] = PatientCommonLib;
+window['_openmrs_esm_patient_common_lib'] = PatientCommonLib;
```
Comment (denniskigen) [inline packages/esm-patient-chart-app/src/index.ts:20]: Must this remain single-quoted? I forget, @ibacher.

File: packages/esm-patient-chart-app/translations/am.json [hunk 5]
```diff
@@ -93,11 +92,13 @@
   "Offline Actions dashboard": "Offline Actions dashboard",
   "openAnyway": "Open anyway",
   "optional": "optional",
-  "orderDurationAndUnit": "for {duration} {durationUnit}",
+  "orderDurationAndUnit": "for {{duration}} {{durationUnit}}",
   "orderIndefiniteDuration": "Indefinite duration",
   "other": "Other",
-  "paginationPageText_one": "of {count} page",
-  "paginationPageText_other": "of {count} pages",
+  "paginationItemsCount_one": "{{pageItemsCount}} / {{count}} items",
```
Comment (denniskigen) [inline packages/esm-patient-chart-app/translations/am.json:98]:
```suggestion
  "paginationItemsCount_one": "{{pageItemsCount}} / {{count}} item",
```

File: packages/esm-patient-chart-app/translations/en.json [hunk 6]
```diff
@@ -93,11 +92,13 @@
   "Offline Actions dashboard": "Offline Actions dashboard",
   "openAnyway": "Open anyway",
   "optional": "optional",
-  "orderDurationAndUnit": "for {duration} {durationUnit}",
+  "orderDurationAndUnit": "for {{duration}} {{durationUnit}}",
   "orderIndefiniteDuration": "Indefinite duration",
   "other": "Other",
-  "paginationPageText_one": "of {count} page",
-  "paginationPageText_other": "of {count} pages",
+  "paginationItemsCount_one": "{{pageItemsCount}} / {{count}} items",
```
Comment (denniskigen) [inline packages/esm-patient-chart-app/translations/en.json:98]:
```suggestion
  "paginationItemsCount_one": "{{pageItemsCount}} / {{count}} item",
```

File: packages/esm-patient-chart-app/translations/fr.json [hunk 7]
```diff
@@ -93,12 +92,15 @@
...
   "openAnyway": "Ouvert dans tous les cas",
   "optional": "optionnel",
-  "orderDurationAndUnit": "pour {duration} {durationUnit}",
+  "orderDurationAndUnit": "pour {{duration}} {{durationUnit}}",
   "orderIndefiniteDuration": "Durée indéfinie",
   "other": "Autre",
-  "paginationPageText_one": "of {count} page",
-  "paginationPageText_many": "of {count} pages",
-  "paginationPageText_other": "of {count} pages",
+  "paginationItemsCount_one": "{{pageItemsCount}} / {{count}} items",
```
Comment (denniskigen) [inline packages/esm-patient-chart-app/translations/fr.json:98]:
```suggestion
  "paginationItemsCount_one": "{{pageItemsCount}} / {{count}} item",
```

File: packages/esm-patient-chart-app/translations/he.json [hunk 8]
```diff
@@ -93,13 +92,17 @@
...
   "optional": "אופציונלי",
-  "orderDurationAndUnit": "למשך {duration} {durationUnit}",
+  "orderDurationAndUnit": "למשך {{duration}} {{durationUnit}}",
   "orderIndefiniteDuration": "למשך זמן בלתי מוגבל",
   "other": "אחר",
-  "paginationPageText_one": "of {count} page",
-  "paginationPageText_two": "of {count} pages",
-  "paginationPageText_many": "of {count} pages",
-  "paginationPageText_other": "of {count} pages",
+  "paginationItemsCount_one": "{{pageItemsCount}} / {{count}} items",
```
Comment (denniskigen) [inline packages/esm-patient-chart-app/translations/he.json:98]:
```suggestion
  "paginationItemsCount_one": "{{pageItemsCount}} / {{count}} item",
```

File: packages/esm-patient-chart-app/src/index.ts (hunk 4 above)
Comment (ibacher) [reply packages/esm-patient-chart-app/src/index.ts:20]: No. This has to be the prettier configuration.

**Review by denniskigen — APPROVED** [review]
LGTM. Thanks a bunch, @vasharma05.

**Pull request merged**